x-pack/plugins/apm/public/components/shared/PropertiesTable/NestedKeyValueTable.tsx

```diff
@@ -56,24 +56,30 @@ const EmptyValue = styled.span`
   color: ${colors.gray3};
 `;
 
-function formatValue(value: any): JSX.Element {
+export function FormattedKey({
+  k,
+  value
+}: {
+  k: string;
+  value: any;
+}): JSX.Element {
+  if (value == null) {
+    return <EmptyValue>{k}</EmptyValue>;
+  }
+
+  return <React.Fragment>{k}</React.Fragment>;
+}
+
+export function FormattedValue({ value }: { value: any }): JSX.Element {
   if (_.isObject(value)) {
     return <pre>{JSON.stringify(value, null, 4)}</pre>;
   } else if (_.isBoolean(value) || _.isNumber(value)) {
-    return <React.Fragment>{String(value)}</React.Fragment>;
+    value = String(value);
```

Comment:
I like to avoid reassigning function params (and actually avoid reassignment in general). 
What about:

```js
return <React.Fragment>{String(value)}</React.Fragment>
```